### ISSUE TITLE
Crush (v3.0.7), editor mode, loaded a save file

### ISSUE THREAD

**Issue by ZwerOxotnik** [lossycrypt/ScreenshotHotkey#1]
```
 434.361 Error MainLoop.cpp:1285: Exception at tick 9741: The mod Eradicator's Screenshot Maker Deluxe 3.0 (3.0.7) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event eradicators-screenshot-maker::on_player_display_resolution_changed (ID 95)
LuaPlayer API call when LuaPlayer was invalid.
stack traceback:
	[C]: in function '__index'
	__eradicators-screenshot-maker__/control/!init.lua:199: in function 'has_player_camera'
	__eradicators-screenshot-maker__/control/!init.lua:313: in function <__eradicators-screenshot-maker__/control/!init.lua:311>
```

**Comment by lossycrypt** [comment]
Can't reproduce. Can you provide steps to reproduce this? Is this perhaps a multiplayer game where you removed some players? I've posted a [hotfix] that probably fixes it, but knowing how you got into that state would be valueable for this and other of my mods.

[hotfix](https://github.com/lossycrypt/eradicators-screenshot-maker/commit/2f6e0ffe7e2b4cf0c9db58921a758be452d79891#diff-9c463e9f5e8b886374b5c7855698e9044d3085df2ffe4b1854861a7ec48bfca0)

**Comment by ZwerOxotnik** [comment]
Oh, I don't get error immediately after loading a save in v3.0.8.

Also, I don't understand why you changed the dependencies.

So, there's another bug now:

```
 324.326 Error MainLoop.cpp:1285: Exception at tick 391: The mod Eradicator's Screenshot Maker Deluxe 3.0 (3.0.8) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event eradicators-screenshot-maker::on_lua_shortcut (ID 35)
LuaPlayer API call when LuaPlayer was invalid.
stack traceback:
	[C]: in function '__index'
	__eradicators-screenshot-maker__/control/ScreenshotGui.lua:205: in function 'destroy_gui'
	__eradicators-screenshot-maker__/control/ScreenshotGui.lua:269: in function 'open'
	__eradicators-screenshot-maker__/control/!init.lua:261: in function 'try_enable_screenshot_mode'
	__eradicators-screenshot-maker__/control/!init.lua:299: in function <__eradicators-screenshot-maker__/control/!init.lua:292>
```

Steps to reproduce the last bug (probably, I did almost the same in v.3.0.7):

1.  Press "Map editor" button
2.  Then "New scenario"
3.  Then "Next"
4.  Then "Play"
5.  Press "esc" key to open menu in the game
6.  Press "Save as" button
7.  Enter "bug" as a new name for save
8.  Press "Quit map editor" button
9.  Then "Map editor"
10.  Then "Edit scenario"
11.  Select the "bug" save and press "play"
12.  Select and press "Toggle screenshot mode" in shortcuts of the player or press "Print screen" key (It's a default key for the GUI in the mod)

**Comment by lossycrypt** [comment]
Thanks for the report, it's now fixed for 3.0.9.  bb19003065ddabf90e1f1bf75633b8c79696bf40

(The dependency versions are automatically updated because I can't test with every factorio version.)
